### PR TITLE
INTER-57: New Tooltip UI

### PR DESCRIPTION
**Pull request by nayanmarlin**
image
image
image
image

### CODE REVIEW

File: src/lib/atoms/v2/tooltips/Tooltip.svelte [hunk 1]
```diff
@@ -1,18 +1,50 @@
...
->
-	<slot />
+<div class="group relative inline-block cursor-pointer text-left" data-testid="tooltip">
+	<slot name="tooltipIcon" />
+	<div
+		class={cn(
+			'absolute z-[999999999] box-border hidden min-w-[320px] rounded-xl bg-white shadow-[0px_12px_16px_-4px_#10182814] group-hover:block',
+			tooltipPosition[placement].container
+		)}
+		style={`min-width: ${minWidth}`}
```
Comment (souvikmishra) [inline src/lib/atoms/v2/tooltips/Tooltip.svelte:39]: is there a specific reason why we are using a separate style attribute to apply `min-width`?

File: src/lib/atoms/v2/tooltips/Tooltip.svelte [hunk 2]
```diff
@@ -1,18 +1,50 @@
...
-	data-testid="tooltip"
-	class="tooltip inline-flex text-left font-normal {tooltipClass}"
-	data-tip={tooltipText}
->
-	<slot />
+<div class="group relative inline-block cursor-pointer text-left" data-testid="tooltip">
+	<slot name="tooltipIcon" />
+	<div
+		class={cn(
+			'absolute z-[999999999] box-border hidden min-w-[320px] rounded-xl bg-white shadow-[0px_12px_16px_-4px_#10182814] group-hover:block',
```
Comment (souvikmishra) [inline src/lib/atoms/v2/tooltips/Tooltip.svelte:36]: maybe avoid using super high z-indices?

File: src/lib/atoms/v2/tooltips/Tooltip.svelte (hunk 1 above)
Comment (nayanmarlin) [reply src/lib/atoms/v2/tooltips/Tooltip.svelte:39]: To add min-width at high level if content is very small. @souvikmishra

File: src/lib/atoms/v2/tooltips/Tooltip.svelte (hunk 2 above)
Comment (nayanmarlin) [reply src/lib/atoms/v2/tooltips/Tooltip.svelte]: Added z-100 @souvikmishra

**Pull request merged**